Source/WebCore/platform/graphics/CodecUtilities.cpp

```diff
@@ -0,0 +1,257 @@
+            break;
+        case AV1ConfigurationLevel::Level_5_2:
+            builder.append(WEB_UI_STRING_KEY("Level 5.2", "Level 5.2 (AV1 Codec Level)", "Codec Strings"));
+            break;
+        case AV1ConfigurationLevel::Level_5_3:
+            builder.append(WEB_UI_STRING_KEY("Level 5.3", "Level 5.3 (AV1 Codec Level)", "Codec Strings"));
+            break;
+        case AV1ConfigurationLevel::Level_6_0:
+            builder.append(WEB_UI_STRING_KEY("Level 6.0", "Level 6.0 (AV1 Codec Level)", "Codec Strings"));
+            break;
+        case AV1ConfigurationLevel::Level_6_1:
+            builder.append(WEB_UI_STRING_KEY("Level 6.1", "Level 6.1 (AV1 Codec Level)", "Codec Strings"));
+            break;
+        case AV1ConfigurationLevel::Level_6_2:
+            builder.append(WEB_UI_STRING_KEY("Level 6.2", "Level 6.2 (AV1 Codec Level)", "Codec Strings"));
+            break;
+        case AV1ConfigurationLevel::Level_6_3:
+            builder.append(WEB_UI_STRING_KEY("Level 6.3", "Level 6.3 (AV1 Codec Level)", "Codec Strings"));
+            break;
+        case AV1ConfigurationLevel::Level_7_0:
+            builder.append(WEB_UI_STRING_KEY("Level 7.0", "Level 7.0 (AV1 Codec Level)", "Codec Strings"));
+            break;
+        case AV1ConfigurationLevel::Level_7_1:
+            builder.append(WEB_UI_STRING_KEY("Level 7.1", "Level 7.1 (AV1 Codec Level)", "Codec Strings"));
+            break;
+        case AV1ConfigurationLevel::Level_7_2:
+            builder.append(WEB_UI_STRING_KEY("Level 7.2", "Level 7.2 (AV1 Codec Level)", "Codec Strings"));
+            break;
+        case AV1ConfigurationLevel::Level_7_3:
+            builder.append(WEB_UI_STRING_KEY("Level 7.3", "Level 7.3 (AV1 Codec Level)", "Codec Strings"));
+            break;
+        }
+        builder.append(")");
+        return builder.toString();
+    }
+
+    if (auto parameters = parseDoViCodecParameters(codecString))
+        return WEB_UI_STRING_KEY("Dolby Vision", "Dolby Vision Codec String", "Codec Strings");
+
+    if (codecString.startsWith("opus"_s))
+        return WEB_UI_STRING_KEY("Opus", "Opus Codec String", "Codec Strings");
+
+    if (codecString.startsWith("vorbis"_s))
+        return WEB_UI_STRING_KEY("Vorbis", "Vorbis Codec String", "Codec Strings");
+
+    if (codecString.startsWith("mp4a.40."_s)) {
+        auto objectType = codecString.substring(8);
+        auto objectValue = parseInteger<uint8_t>(objectType, 10);
+
+        switch (objectValue.value_or(0)) {
+        case 1:
+        case 3:
+        case 4:
+        case 5:
+        case 6:
+        case 29:
+            return WEB_UI_STRING_KEY("MPEG-4 AAC", "MPEG-4 AAC Codec String", "Codec Strings");
+        case 2:
+            return WEB_UI_STRING_KEY("AAC-LC", "AAC-LC Codec String", "Codec Strings");
+        case 34:
+            WEB_UI_STRING_KEY("MP3", "MP3 Codec String", "Codec Strings");
+        }
```

Comment:
No handling of the default value `0`?

Reply:
No, it should fall through to "MP4 Audio".